docs/12.EXT-REFERENCE-MODULE.md

```diff
@@ -150,13 +154,31 @@ jerry_value_t js_module = jerryx_module_resolve (requested_module, resolvers, 2)
 
 # Module helper macros
 
+## JERRYX_NATIVE_MODULE_INIT
+
+**Summary**
+
+Expands to the initializer of a variable of type `jerryx_native_module_t`.
+
+**Prototype**
+
+```c
+#define JERRYX_NATIVE_MODULE_INIT(module_name, on_resolve_cb)
```

Comment:
I don't really like that we will have a JERRYX_NATIVE_MODULE_INIT and JERRYX_NATIVE_MODULE macros, which both inits something in different scenarios. The name of both should be more explicit about what they actually do.